.dockerignore

```diff
@@ -0,0 +1,8 @@
+.env
+.env.template
+.env.test
+docker.env
+.gitignore
+.vagrant
+pytest.ini
+test_*
```

Comment:
This is a reasonable list, but you should also add `Dockerfile` and `.git`

Later, when we come to running tests inside Docker, you will not want to ignore `.env.test` or `test_*`